R/utils.R

```diff
@@ -1,12 +1,16 @@
 #' Utility function to calculate relative abundances
 #'
+#'
```

Comment:
Remove non-functional changes such as lines containing only `#'`, to simplify review.  In general if there are formatting or stylistic changes to make, it would be better to do those as their own separate PR.

Reply:
Okay, I'll make the changes

Reply:
Done